### PR TITLE
adding robotics blog

### PR DESCRIPTION
**Pull request by MartyIce**

### CODE REVIEW

File: _posts/2024-11-14-robotics1.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+layout: post
+title: Robotics
+tweetText: Robotics
```
Comment (MarkMavis26) [inline _posts/2024-11-14-robotics1.md:4]: hey name?

**Review by MarkMavis26 — APPROVED** [review]
looks good

File: _posts/2024-11-14-robotics1.md (hunk 1 above)
Comment (MartyIce) [reply _posts/2024-11-14-robotics1.md:4]: I like this name

Comment (MartyIce) [reply _posts/2024-11-14-robotics1.md:4]: This git page says:

https://git-scm.com/downloads/win

**Pull request merged**